test/operator.jl

```diff
@@ -648,13 +648,19 @@ function operators_test(ModelType::Type{<:JuMP.AbstractModel}, VariableRefType::
     end
 
     @testset "operator_warn" begin
-        m = Model()
+        m = ModelType()
         @variable m x[1:51]
-        @test m.operator_counter == 0
+        # JuMPExtension does not have the operator_counter but this test
+        # check that it does not make it fail
```

Comment:
Awkward english. Just say that "JuMPExtension does not have the operator_counter field."?